src/sentry/static/sentry/app/views/issueList/overview.tsx

```diff
@@ -401,13 +404,16 @@ class IssueListOverview extends React.Component<Props, State> {
       const requestParams: CountsEndpointParams = {
         ...omit(endpointParams, 'query'),
         // fetch the counts for the tabs whose counts haven't been fetched yet
-        query: TabQueriesWithCounts.filter(_query => _query !== currentTabQuery),
+        query: TabQueriesWithCounts.filter(
+          _query => _query !== currentTabQuery || _query === Query.REPROCESSING
```

Comment:
Why is the reprocessing query special?

Reply:
We only want to display the reprocessing tab if there is at least one event being reprocessed. Therefore, whenever we send a request to get the queryCount, we also want to know the reprocessing count. This logic is not applied only in case the current tab is Reprocessing